src/Discord/CommandClient/Command.php

```diff
@@ -163,6 +190,17 @@ public function handle(Message $message, array $args)
             array_unshift($args, $subCommand);
         }
 
+        $currentTime = round(microtime(true));
```

Comment:
Use `$currentTime = round(microtime(true) * 1000)` to get the current epoch in milliseconds.